sbncode/SBNEventWeight/jobs/flux/eventweight_flux_sbn.fcl

```diff
@@ -0,0 +1,232 @@
+	PositiveSystematicVariation_hist_file: "beamData/UnisimHists/may06_nucleoninexsec_up_rgen610.6_flux.root"
+	NegativeSystematicVariation_hist_file: "beamData/UnisimHists/may06_nucleoninexsec_down_rgen610.6_flux.root"
+  }
+
+  nucleonqexsec: {
+	type: Flux
+	random_seed: 10
+	mode: multisim
+	number_of_multisims: 1000
+	parameter_list: ["nucleonqexsec"]
+	
+	calc_type: "Unisim"
+	scale_factor_pos: 0.333 #From MiniBooNE 
+	scale_factor_neg: 0.573 #From MiniBooNE
+	CentralValue_hist_file: "beamData/UnisimHists/may06_10kpot_ntrd1000_flux.root"
+	PositiveSystematicVariation_hist_file: "beamData/UnisimHists/may06_nucleonqexsec_up_rgen610.6_flux.root"
+	NegativeSystematicVariation_hist_file: "beamData/UnisimHists/may06_nucleonqexsec_down_rgen610.6_flux.root"
+  }
+
+  nucleontotxsec: {
+	type: Flux
+	random_seed: 9
+	mode: multisim
+	number_of_multisims: 1000
+	parameter_list: ["nucleontotxsec"]
+	
+	calc_type: "Unisim"
+	scale_factor_pos: 1  
+	scale_factor_neg: 1
+	CentralValue_hist_file: "beamData/UnisimHists/may06_10kpot_ntrd1000_flux.root"
+	PositiveSystematicVariation_hist_file: "beamData/UnisimHists/may06_nucleontotxsec_up_rgen610.6_flux.root"
+	NegativeSystematicVariation_hist_file: "beamData/UnisimHists/may06_nucleontotxsec_down_rgen610.6_flux.root"
+  }
+
+
+  kplus: {
+    type: Flux
+    random_seed: 4
+    mode: multisim
+    number_of_multisims: 1000
+    parameter_list: ["kplus"]
+    parameter_sigma: [1] 
+
+    calc_type: "PrimaryHadronFeynmanScaling"
+    PrimaryHadronGeantCode: [321] # geant code for particle production on Be
+    scale_factor_pos: 1
+    ExternalData: "beamData/ExternalData/BNBExternalData_uBooNE.root"
+    #ExternalFit: "empty"
+   }
+
+  kminus: {
+    type: Flux 
+    random_seed: 6
+    mode: multisim
+    number_of_multisims: 1000
+    parameter_list: ["kminus"]
+    parameter_sigma: [1]
+
+	calc_type: "PrimaryHadronNormalization"
+    PrimaryHadronGeantCode: [-321] # geant code for particle production on Be
+    scale_factor_pos: 1
+   } 
+
+  kzero: {
+    type: Flux
+    random_seed: 5
+    mode: multisim
+    number_of_multisims: 1000
+    parameter_list: ["kzero"]
+    parameter_sigma: [1]
+
+    calc_type: "PrimaryHadronSanfordWang"
+    PrimaryHadronGeantCode: [130, 310, 311] # geant code for particle production on Be
+    scale_factor_pos: 1
+    ExternalData: "beamData/ExternalData/BNBExternalData_uBooNE.root"
+   }
+
+
+  piplus: {
+    type: Flux
+    random_seed: 2
+    mode: multisim
+    number_of_multisims: 1000
+    parameter_list: ["piplus"]
+    parameter_sigma: [1]
+
+    calc_type: "PrimaryHadronSWCentralSplineVariation"
+    PrimaryHadronGeantCode: [211] # geant code for particle production on Be
+    scale_factor_pos: 1
+    ExternalData: "beamData/ExternalData/BNBExternalData_uBooNE_SplinesHARP.root"
+    ExternalFit: "beamData/ExternalData/BNBExternalData_uBooNE.root"
+   }
+
+  piminus: {
+    type: Flux
+    random_seed: 3
```

Comment:
how random are these seeds?

Reply:
This seed is what was used in MicroBooNE, and it uses the same  `CLHEP/Random/RandGaussQ.h` to generate random numbers.